src/components/NewGalaxyModal.js

```diff
@@ -286,13 +285,24 @@ export const NewGalaxyModal = _.flow(
   }
 
   const renderPersistentDiskSection = () => {
-    const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '1rem', alignItems: 'center' }
     return div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
       div({ style: styles.headerText }, ['Persistent disk']),
       div({ style: { marginTop: '0.5rem' } }, [
         'Persistent disks store analysis data.'
         // TODO Add info on PDs for Galaxy (similarly to in NewRuntimeModal.js)
       ]),
+      renderPersistentDiskSizeSection()
+    ])
+  }
+
+  const renderPersistentDiskSizeSection = () => {
```

Comment:
Not sure if this logic is the best way to do this - could use some feedback

Reply:
What logic exactly were you unsure about?

Reply:
the ternary in `renderPersistentDiskSizeSection` - justin addressed this in a comment below and i will try out using a `Utils` method to clean this up a bit